### ISSUE TITLE
Know which Widgets have been used.

### ISSUE THREAD

**Issue by holmwood** [elementor/elementor#8343]
Most Elementor users will activate all the Elementor Addin plugin they have, before starting to design.

Once the website is finished Elementor should show the users which add-ons have not been used.  So they can be deactivated - to improve site speed optimization.

Do this either by changing the color of the used widgets or have a separate section showing the widgets and the plugin from with they originated.  The best would be just a list of Elementor Add-ons plugins that can be safely deleted as they have not been used.

**Comment by holmwood** [comment]
**Re-written, but without English errors:**

Most Elementor users will activate all the Elementor add-on plugins they have before they starting to design.

Once the website is finished, Elementor should show the user which add-ons have not been used. So they can be deactivated - to improve site speed optimization.

Do this either by changing the color of the used widgets or have a separate section showing the widgets and the plugin from with they originated. **The best way would be just a list of Elementor add-on's plugins that can be safely deleted, as they have not been used.**

**Comment by shilo-ey** [comment]
Hi @holmwood 

Elementor provides the "core engine" for 3rd party add-ons, which means they can communicate with other layers of the page builder natively.

When a developer creates an add-on, we can't take responsibility and maintain it. the author himself needs to do it, so we can't really help in this case.

Thanks!

**Comment by holmwood** [comment]
When I posted this feature request on Facebook to the CMO of Elementor, it received more likes than another Elementor feature request.No one is asking you to take 'responsibility' or 'maintain' anything – heaven forbid.Just track and report on which widgets have actually been used in the production of an Elementor page.  So if it is not your problem, I guess we will have to wait until it is. Best regards, Chris  17.06.2019, 14:02, "shilo-ey" <notifications@github.com>:Hi @holmwoodElementor provides the "core engine" for 3rd party add-ons, which means they can communicate with other layers of the page builder natively.When a developer creates an add-on, we can't take responsibility and maintain it. the author himself needs to do it, so we can't really help in this case.Thanks!—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by holmwood** [comment]
This is my post that received all the likes:Ben Pines tell us Elementor will soon show us what addon plugins we did not use.Most Elementor users will activate all the Elementor addon plugins they have before they start to design.Once the website is finished, Elementor should show the user which Elementor addon plugins have not been used in the design. So they can then be deactivated and deleted - to improve site speed optimization.  19.06.2019, 08:48, "chris@viptaxipro.com" <chris@viptaxipro.com>: When I posted this feature request on Facebook to the CMO of Elementor, it received more likes than another Elementor feature request.No one is asking you to take 'responsibility' or 'maintain' anything – heaven forbid.Just track and report on which widgets have actually been used in the production of an Elementor page.  So if it is not your problem, I guess we will have to wait until it is. Best regards, Chris  17.06.2019, 14:02, "shilo-ey" <notifications@github.com>:Hi @holmwoodElementor provides the "core engine" for 3rd party add-ons, which means they can communicate with other layers of the page builder natively.When a developer creates an add-on, we can't take responsibility and maintain it. the author himself needs to do it, so we can't really help in this case.Thanks!—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.
